### PR TITLE
Expand descriptions of CoffeeScript classes

### PR DESCRIPTION
**Pull request by malandrina**
- https://trello.com/c/6labM2my

### CODE REVIEW

File: book/application_development/plot_points_on_a_map.md [hunk 1]
```diff
@@ -25,16 +25,18 @@ application layout body:
 
 ` app/views/layouts/application.html.erb@01d053a:10,16
 
-And now we get to the fun part: Creating a `Mapper` to handle drawing and updating
-the map.
+Now that our initial setup is complete, we create some CoffeeScript classes to
```
Comment (joshuaclayton) [inline book/application_development/plot_points_on_a_map.md]: "Now that our initial setup is complete" => "With our initial set-up complete"?

File: book/application_development/plot_points_on_a_map.md [hunk 2]
```diff
@@ -25,16 +25,18 @@ application layout body:
...
+handle interactions with the maps API. We begin by creating a `Mapper` to append
+the map to the DOM and set the map bounds.
```
Comment (joshuaclayton) [inline book/application_development/plot_points_on_a_map.md]: "to append the map to the DOM and set the map bounds" => "to display the map on the page with the correct coordinates"? Trying to make this a bit more based on behavior than implementation, which can be inferred from the code itself.

File: book/application_development/plot_points_on_a_map.md [hunk 3]
```diff
@@ -25,16 +25,18 @@ application layout body:
...
 
 ` app/assets/javascripts/mapper.coffee@5bb8973
 
-We also create a `MapBounds` class:
+Next we create a `MapBounds` class which allows us to encapsulate the
+behavior of the Google `LatLngBounds` function.
```
Comment (joshuaclayton) [inline book/application_development/plot_points_on_a_map.md]: "which provides a simple interface for interacting with Google's representation of coordinates and bounds"? Again, leaning towards the code explain what's going on and the text discuss more behavior. Definitely interested in your thoughts on this!

File: book/application_development/plot_points_on_a_map.md [hunk 4]
```diff
@@ -25,16 +25,18 @@ application layout body:
...
 
 ` app/assets/javascripts/map_bounds.coffee@5bb8973
 
-... and a `Map` class:
+We also create a `Map` class to wrap the Google `Map` function.
```
Comment (joshuaclayton) [inline book/application_development/plot_points_on_a_map.md]: "We also create a `Map` class to provide a simple interface to the Google Maps JavaScript API for rendering a responsive map with the correct points plotted."?

**Comment by malandrina** [comment]
@joshuaclayton, great suggestions! Describing the behavior rather than the implementation of these classes is definitely the right approach. Changed in https://github.com/thoughtbot/geocoding_on_rails/commit/d5ff5232e4dbb6d1cc2aa5d6e54bfd5ee7230594.

**Comment by joshuaclayton** [comment]
:+1: looks great!

**Pull request merged**